Gems/LyShine/Code/Source/UiCanvasManager.cpp

```diff
@@ -915,60 +912,56 @@ AZ::EntityId UiCanvasManager::LoadCanvasInternal(const AZStd::string& assetIdPat
     // editor version so that the user can test their canvas without saving it
     UiCanvasComponent* canvasComponent = FindEditorCanvasComponentByPathname(assetIdPath);
 
-    // This scope opened here intentionally to control the lifetime of the AZ_ASSET_NAMED_SCOPE
+    if (canvasComponent)
     {
```

Comment:
Super nit-pick: Definitely not worth changing back, but in situations like this where it's a really large change I think it's maybe worth keeping the empty scope even if it's not necessary anymore, simply for the sake of making the diff more readable (not just for review but for anyone looking back at the history).

Reply:
Sure, I just didnt want to leave the double scope around